timestamp-batch/src/main/java/io/spring/configuration/TimestampBatchTaskConfiguration.java

```diff
@@ -50,18 +58,46 @@
 	@Autowired
 	private TimestampBatchTaskProperties config;
 
+	/**
+	 * Override default transaction isolation level 'ISOLATION_REPEATABLE_READ' which Oracle does not
+	 * support.
+	 */
+	@Configuration
+	@ConditionalOnProperty(value = "spring.datasource.driver", havingValue = "oracle.jdbc.OracleDriver")
+	static class OracleBatchConfig {
+		@Bean
+		BatchConfigurer oracleBatchConfigurer(DataSource dataSource) {
```

Comment:
Is bean overriding enabled or is there no `BatchConfigurer` already configured, or is the auto-configured one respectful  w/ `@ConditionalOnMissingBean`?

Reply:
Must be B by process of elimination. I haven't looked at that in BatchAutoConfig, but rolling your own BC is the way it is done.

Reply:
I thought so too, and [voila](https://github.com/spring-projects/spring-boot/blob/fac2430dff0d5e00b4a333b80205bac14a1caf75/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/batch/BatchConfigurerConfiguration.java#L40). Note that the JpaBatchConfigurer will be omitted as well, in this case does not matter though.